app/src/main/java/woowacourse/movie/ui/alarm/ReservationAlarmReceiver.kt

```diff
@@ -15,10 +15,20 @@ class ReservationAlarmReceiver : BroadcastReceiver() {
     ) {
         val notificationManager = NotificationCreator(context)
         val movie: MovieTicketModel =
-            intent.getSerializableExtraCompat(SettingFragment.KEY_MOVIE)
+            intent.getSerializableExtraCompat(MovieDetailActivity.KEY_MOVIE)
                 ?: return
 
         notificationManager.createNotificationChannel()
-        notificationManager.makeNotification(context, movie)
+        notificationManager.makeNotification(movie)
+    }
+
+    companion object {
+        fun getIntent(ticketModel: MovieTicketModel, context: Context): Intent {
+            val intent = Intent(context, this::class.java).apply {
```

Comment:
```suggestion
            val intent = Intent(context, ReservationAlarmReceiver::class.java).apply {
```
companion object 내에서 this를 사용하게 되면, ReservationAlarmReceiver가 아닌 ReservationAlarmReceiver.Companion 으로 접근돼요.